annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,3 +1,4 @@
+spring.main.web-application-type=none
```

Comment:
I still think we should handle this differently. Not the best user experience to have their pipeline crash after an upgrade. But ok to leave as is. Filed a separate issue for this: https://github.com/genome-nexus/genome-nexus-annotation-pipeline/issues/168

Reply:
Will be spending more time on the build issue today so I'll take a look at this as well